java/test/jmri/util/junit/rules/RetryRuleTest.java

```diff
@@ -20,19 +20,23 @@ public void testFirstOK() {
 
     @Test
     public void testPassOnThirdRetry() {
-        if (countPassOnThirdRetry++ < 3) {
+        countPassOnThirdRetry++;
+        if ( countPassOnThirdRetry < 4 ) {
```

Comment:
Why this change? I don't see how it matters?

Reply:
It shows as a minor warning in Netbeans, also I find the logic a lot easier to read through.
Should the test fail on the 3rd or 4th pass ?